src/utils/msgCollector.ts

```diff
@@ -0,0 +1,14 @@
+import { CommandInteraction, Message } from 'discord.js';
+
+export function awaitMessage(interaction: CommandInteraction) {
+	const filter = (msg: Message) => interaction.user.id === msg.author.id;
+
+	const sendedMessage = interaction.channel!.awaitMessages({ max: 1, filter }).then(msg => {
+		return {
+			content: msg.first()!.content.toLocaleLowerCase().trim() ?? '',
```

Comment:
Shouldn't this non-null-assertion be replaced by the optional chaining operator? Since we're using the nullish coalescing operator.
```suggestion
			content: msg.first()?.content.toLocaleLowerCase().trim() ?? '',
```

Reply:
True, will refactor.